tests/integration/test_config_hidden_attributes/configs/config.xml

```diff
@@ -0,0 +1,11 @@
+<clickhouse>
+    <max_table_size_to_drop hidden="true">60000000000</max_table_size_to_drop>
+    <max_partition_size_to_drop hidden="false">40000000000</max_partition_size_to_drop>
```

Comment:
The code uses `Poco::NumberParser::parseBool(element.getAttribute("hidden"))`. It would be nice to validate that `hidden = "1"` achieves the same result (just to avoid surprises for people who prefer "1" over "true").

Reply:
Done. Added test for "1" value.